docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,128 @@
+- **Choose a script type:** `Javascript`
+- **What do you want to name your add-in?** `My SSO Office Add-in`
+- **Which Office client application would you like to support?** `Excel`
+
+![A screenshot of the prompts and answers for the Yeoman generator](../images/yo-office-sso-excel.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+[!include[Yeoman generator next steps](../includes/yo-office-next-steps.md)]
+
+## Explore the project
+
+The add-in project that you've created with the Yeoman generator contains code for an SSO-enabled task pane add-in.
+
+- The **./ENV** file in the root directory of the project defines constants that are used by the add-in project.
+- The **./manifest.xml** file in the root directory of the project defines the settings and capabilities of the add-in.
+- The **./src/helpers/documentHelper.js** file uses the Office JavaScript library to add the data from Microsoft Graph to the Office document.
+- The **./src/helpers/fallbackauthdialog.html** file is the UI-less page that loads the fallback authentication method's JavaScript.
+- The **./src/helpers/fallbackauthdialog.js** file contains the fallback authentication method's JavaScript that signs on the user with msal.js.
+- The **./src/helpers/fallbackauthhelper.js** file contains the task pane JavaScript that invokes the fallback authentication method in scenarios when SSO authentication is not supported.
+- The **./src/helpers/ssoauthhelper.js** file contains the JavaScript call to the SSO API, `getAccessToken`, receives the bootstrap token, initiates the swap of the bootstrap token for an access token to Microsoft Graph, and calls to Microsoft Graph for the data.
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane.
+- The **./src/taskpane/taskpane.css** file contains the CSS that's applied to content in the task pane.
+- The **./src/taskpane/taskpane.js** file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office host application.
+
+## Configure SSO
+
+At this point, your add-in project has been created and contains the code that's necessary to facilitate the SSO process. Next, complete the following steps to configure SSO for your add-in.
+
+1. Navigate to the root folder of the project.
+
+    ```command&nbsp;line
+    cd "My SSO Office Add-in"
+    ```
+
+2. Run the following command to configure SSO for the add-in.
+
+    ```command&nbsp;line
+    npm run configure-sso
+    ```
+
+    > [!WARNING]
+    > This command will fail if your tenant is configured to require two-factor authentication. In this scenario, you'll need to manually complete the Azure app registration and SSO configuration steps manually, as described in the [Create a Node.js Office Add-in that uses single sign-on](../develop/create-sso-office-add-ins-nodejs.md) tutorial.
```

Comment:
You say manually twice, maybe "you'll need to manually complete the ... configuration steps, as described in the.."